### PR TITLE
Chapter 6

### PR DESCRIPTION
**Pull request by muizzuddinyafiq**
ready

### CODE REVIEW

**Comment by muizzuddinyafiq** [comment]
ready

File: app/assets/stylesheets/users.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/users.scss:1]: delete this file

File: app/helpers/users_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```
Comment (ngocvt-0484) [inline app/helpers/users_helper.rb:1]: delete this

File: test/models/user_test.rb [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+  test "name should be present" do
+    @user.name = " "
+    assert_not @user.valid?
+  end
+
+  test "email should be present" do
+    @user.email = " "
+    assert_not @user.valid?
+  end
```
Comment (ngocvt-0484) [inline test/models/user_test.rb:22]: dont need do test. please delete all test file

**Comment by muizzuddinyafiq** [comment]
ready

**Review by ghost — COMMENTED** [review]
You have 2 commits, please combine into 1

**Comment by muizzuddinyafiq** [comment]
ready

File: config/routes.rb [hunk 4]
```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  get 'users/new'
```
Comment (thaodtd-0665) [inline config/routes.rb]: Recommend use resources
```suggestion
  resources :users
```

**Comment by muizzuddinyafiq** [comment]
ready

**Review by thaodtd-0665 — APPROVED** [review]

**Comment by thaodtd-0665** [comment]
merged

**Pull request merged**